README.md

```diff
@@ -51,6 +51,7 @@ The SAF CLI is the successor to [Heimdall Tools](https://github.com/mitre/heimda
   * [View](#view) - Identify overall security status and deep-dive to solve specific security defects
   * [Validate](#validate) - Verify pipeline thresholds
   * [Generate](#generate) - Generate InSpec validation code, set pipeline thresholds, and generate options to support other saf commands.
+  * [Supplement](#supplement) - Supplement elements that provide contextual information in an HDF file
```

Comment:
Do you want to add `such as passthrough or target` to this description as well? Up to you.

Reply:
No since I wanted the table of contents' blurb to be shorter